docs_src/pipelinesetup.md

```diff
@@ -1,49 +1,47 @@
-# Pipeline Setup
+# Set up Pipeline
 
-## Step 1: Clone the repository
+### Step 1: Clone the repository
 
 ```
 git clone  https://github.com/intel-retail/automated-self-checkout.git && cd ./automated-self-checkout
 ```
 
-## Step 2: Build Benchmark Containers
+### Step 2: Build the benchmark containers
 
-For benchmarking supported use cases you will need to build the benchmark containers
+For use cases that support benchmarking, build the benchmark containers:
 
 ```bash
 cd benchmark-scripts
 make build-all
 ```
 
-## Step 3: Download Models Manually (Optional)
+### Step 3: Download the models manually (Optional)
 
-Model downloader script is automatically called as part of docker-run.sh.  User can also download the models manually using the model downloader script shown as follows:
+When `docker-run.sh` the Model downloader script is automatically called. You can also download the models manually using the model downloader script:
 
 ```bash
 sh modelDownload.sh
 ```
 
-!!! note
-    This may take a while as there are several models to be downloaded for the first time.
+**_Note:_**  This step might take a while as the script will download several models for the first time.
 
-!!! note
-    To manually download models you can follow links provided in the [Model List](../configs/models/2022/models.list.yml)
+**_Note:_**  For the list of pre-trained models, refer to the [Model List](https://github.com/intel-retail/automated-self-checkout/blob/1.5.0/configs/dlstreamer/models/2022/models.list.yml).
```

Comment:
Will need to update this link to point to main

Reply:
using the relative path should work ../configs/dlstreamer/models/2022/models.list.yml